guide/xml/portgroup-python.xml

```diff
@@ -286,4 +286,21 @@
       </varlistentry>
     </variablelist>
   </section>
+  <section xml:id="reference.portgroup.python.test">
+    <title>python Port test tips</title>
+
+    <para>To add a test import of the built package, the following template may be useful</para>
+
+    <programlisting>if {${name} ne ${subport}} {
+<!--     -->
+<!--     -->    pre-test {
+<!--     -->        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
+<!--     -->    }
+<!--     -->
+<!--     -->    test.run        yes
+<!--     -->    test.cmd        ${python.bin} -c '${python.rootname}'
+<!--     -->    test.target
+<!--     -->
+<!--     -->}</programlisting>
```

Comment:
Probably this should at least mention that a `./setup.py test` target should usually be preferred, since it would run actual unit tests instead of only checking whether the import works. Not a blocker, can also be added later here.